src/app/loginComponents/requests/organization-request-confirm-dialog.html

```diff
@@ -0,0 +1,15 @@
+<h1 mat-dialog-title>{{data.approve ? 'Approve' : 'Decline'}} Organization</h1>
```

Comment:
I think you should say `reject` here. :)

Sorry I wasn't clearer in my earlier suggestion -- I think you should decline invites, but I think you should reject organizations. Because rejecting an org is a bigger deal, I think it needs a stronger word.

Reply:
Oops I'll change those back.